rules/index.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  arrayFunc: require('./array-func'),
+  eslint: require('./eslint'),
+  eslintComments: require('./eslint-comments'),
+  immutable: require('./immutable'),
+  jest: require('./jest'),
+  overrides: require('./overrides'),
+  simpleImportSort: require('./simple-import-sort'),
+  typescriptEslint: require('./typescript'),
+  unicorn: require('./unicorn'),
+};
```

Comment:
Instead of this export, we should list directory and require those files in `index.spec.ts`, so it's less error prone. More importantly, we don't test overrides in any way, which means we don't have e2e tests. Should we create an issue for e2e tests?

Reply:
fixed with: https://github.com/ridedott/eslint-config/pull/319/commits/4da75bac8dd12c5bc982f63f3e10b164437a9892

Reply:
I think it would be good to create issue for e2e